code/modules/projectiles/tgprojectile.dm

```diff
@@ -0,0 +1,1261 @@
+//Some debug variables. Comment/Uncomment them to Enable/Disable the debug messages.
+//#define DEBUG_HIT_CHANCE	0
+//#define DEBUG_HUMAN_DEFENSE	0
+//#define DEBUG_XENO_DEFENSE	0
+//#define DEBUG_CREST_DEFENSE	0
+
+// check _DEFINES/projectiles.dm for some more defines
+
+// TESTING and DEBUGGING and PROFILING equipment
+
+/obj/machinery/pewpew
+	name = "pewpewpew"
+	use_power = NO_POWER_USE
+	var/ammo_left = 800
+	var/datum/ammo/ammo = new /datum/ammo/bullet/pistol
+
+/obj/machinery/pewpew/Initialize()
```

Comment:
Good for testing but don't forget to remove it once the PR is done.

Reply:
This is actually a hidden meme weapon, admins can spawn it somewhere and it will obliterate anyone standing in it's path. The PR won't be done, as I can't optimize it, so it won't be merged :'(